packages/plugin-e2e/src/models/TimeRange.ts

```diff
@@ -0,0 +1,47 @@
+import { Expect } from '@playwright/test';
+import { PluginTestCtx, TimeRangeArgs } from '../types';
+import { GrafanaPage } from './GrafanaPage';
+
+export class TimeRange extends GrafanaPage {
+  constructor(ctx: PluginTestCtx, expect: Expect<any>) {
+    super(ctx, expect);
+  }
+
+  async set({ from, to, zone }: TimeRangeArgs) {
+    try {
+      await this.ctx.page
+        .getByLabel(this.ctx.selectors.components.PanelEditor.General.content)
+        .locator(`selector=${this.ctx.selectors.components.TimePicker.openButton}`)
+        .click({ force: true, timeout: 2000 });
+    } catch (e) {
+      // seems like in older versions of Grafana the time picker markup is rendered twice
+      await this.ctx.page.locator('[aria-controls="TimePickerContent"]').last().click();
```

Comment:
Could it be that the error is caused by something else?

Reply:
Unlikely, but maybe. Anyway, I don't like this so I'll add a `todo` for finding a better solution for this. Have a bunch of todos throughout the code...most of them involves changes in Grafana ui so would like to collect a few before I get to them.